README.md

```diff
@@ -0,0 +1,94 @@
+# gr-stellarstation: GNURadio blocks for interacting with StellarStation
+
+gr-stellarstation is an out-of-tree GNURadio module that provides blocks for easily integrating with the [StellarStation](https://www.stellarstation.com/) API in GNURadio flowgraphs.
+
+## Installation
+
+### PyBOMBS
+
+PyBOMBs support is currently under construction.
+
+### Source Installation
+
+gr-stellarstation, like most OOT modules, can be built from source using CMake.
+
+First, we need to install gr-stellarstation's sole dependency, [Conan](https://conan.io/). The simplest way to do this is through `pip`:
+
+```bash
+$ pip install conan
+```
+
+For alternative methods to install Conan, see the [Conan documentation](https://docs.conan.io/en/latest/installation.html).
+
+After installing Conan, run the following commands to add the relevant Conan recipes:
+
+```bash
+$ conan remote add bincrafters https://api.bintray.com/conan/bincrafters/public-conan
+$ conan remote add inexorgame https://api.bintray.com/conan/inexorgame/inexor-conan
+$ conan remote add stellarstation https://api.bintray.com/conan/infostellarinc/stellarstation-conan
+```
+
+Proceed to install gr-stellarstation normally with CMake:
+
+```bash
+$ git clone https://github.com/infostellarinc/gr-stellarstation.git
+$ cd gr-stellarstation
+$ mkdir build
+$ cmake ..
```

Comment:
Cmake failed dependency errors of cppuni and Doxygen. 
If preparation is required for the build,  README should comments about it..